syntax = "proto3";

package milvus.proto.cgo;
option go_package="github.com/milvus-io/milvus/internal/proto/cgopb";

import "schema.proto";

message LoadIndexInfo {
  int64 collectionID = 1;
  int64 partitionID = 2;
  int64 segmentID = 3;
  schema.FieldSchema field = 5;
  bool enable_mmap = 6;
  string mmap_dir_path = 7;
  int64 indexID = 8;
  int64 index_buildID = 9;
  int64 index_version = 10;
  map<string, string> index_params = 11;
  repeated string index_files = 12;
  string uri = 13;
  int64 index_store_version = 14;
  int32 index_engine_version = 15;
  int64 index_file_size = 16;
}
